h1st/model/remote_model.py

```diff
@@ -0,0 +1,157 @@
+from typing import Dict, Tuple
+import requests
+from .predictive_model import PredictiveModel
```

Comment:
we should use the full path, for example `h1st.model.predictive_model` instead of this relative path